db/model/tenant.go

```diff
@@ -364,7 +364,7 @@ type TenantServiceEnvVar struct {
 	ContainerPort int    `gorm:"column:container_port" validate:"container_port|numeric_between:1,65535" json:"container_port"`
 	Name          string `gorm:"column:name;size:1024" validate:"name" json:"name"`
 	AttrName      string `gorm:"column:attr_name;size:1024" validate:"env_name|required" json:"attr_name"`
-	AttrValue     string `gorm:"column:attr_value;size:2048" validate:"env_value|required" json:"attr_value"`
+	AttrValue     string `gorm:"column:attr_value;size:65535" validate:"env_value|required" json:"attr_value"`
```

Comment:
MySQL row sizes max 65535，so can not set one column max size 65535， should use `text`.